src/cli/login.py

```
@@ -101,6 +105,10 @@ class UrlValidator(pydantic.BaseModel):
     # Login through device code flow
     if args.method == 'code':
         service_client.login_manager.device_code_login(url, args.device_endpoint)
+        try:
+            agent_skill_installer.prompt_skill_installation()
+        except Exception:  # pylint: disable=broad-except
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)